configure.ac

```diff
@@ -227,11 +227,13 @@ AM_CONDITIONAL(USE_DRM, test "$USE_DRM" = "yes")
 
 # Check for X11
 USE_X11="no"
-if test "$enable_x11" = "yes"; then
-    USE_X11="yes"
-    PKG_CHECK_MODULES([X11],    [x11],    [:], [USE_X11="no"])
-    PKG_CHECK_MODULES([XEXT],   [xext],   [:], [USE_X11="no"])
-    PKG_CHECK_MODULES([XFIXES], [xfixes], [:], [USE_X11="no"])
+if test "x$enable_x11" != "xno"; then
+    PKG_CHECK_MODULES([X11],    [x11 xext xfixes],    [USE_X11="yes"], [:])
```

Comment:
`XEXT` and `XFIXES` will now be included in the `X11_CFLAGS`.  Thus, `XEXT_CFLAGS` and `XFIXES_CFLAGS` need to be removed from `AM_CPPFLAGS` in `va/x11/Makefile.am` since they will no longer be defined.